horace_core/admin/cache.py

```diff
@@ -146,7 +141,7 @@ def process_file(argi):
     # Start parallel jobs:
     job_list = [];
     #read_chunk(filename,chunk_beg[0],chunk_size[0],buf_size,True,n_threads)
-    for nthr in xrange(n_threads):
+    for nthr in range(n_threads):
```

Comment:
This should be done with a `map` and `with pool` not a loop

Reply:
Do not think so. This is the procedure for single thread .